opencodelists/management/commands/import_mapping_data.py

```diff
@@ -7,10 +7,17 @@
 
 class Command(BaseCommand):
     def add_arguments(self, parser):
-        parser.add_argument("dataset", choices=mappings.keys())
-        parser.add_argument("release_dir")
+        parser.description = "Import a coding system mapping"
+        parser.add_argument("mapping", choices=mappings.keys(), help="Mapping name")
+        parser.add_argument(
+            "source",
+            help=(
+                "File or dir that defines the mapping, see import_data.py in "
```

Comment:
nit: maybe the per-mapping README is a friendlier guide than going to the source?

Reply:
Good point, thanks. I'll mention both I think, in case there is no README or it's inadequate.